packages/query/types/inscription.ts

```diff
@@ -24,3 +24,37 @@ export interface InscriptionResponseHiro {
   tx_id: string;
   value: string;
 }
+
+interface BestinSlotInscriptionDelegateResponse {
+  delegate_id: string;
+  render_url?: string | null;
+  mime_type?: string | null;
+  content_url: string;
+  bis_url: string;
+}
+
+interface BestInSlotInscriptionMetadataResponse {
+  name: string;
+  [key: string]: any;
+}
+
+export interface BestinslotInscriptionResponse {
```

Comment:
IMO this should live in the query where we pull this response. Also can make this from the schema definition with `zod.Infer`